7Cav-Alive-WFW.Altis/scripts/vehicle/fn_respawnVehicleInitialize.sqf

```diff
@@ -44,3 +44,12 @@ if (!isNil "_vehicleInit") then
 };
 
 [_vehicle] spawn JB_RV_Monitor;
+
```

Comment:
You don't need to have the code in both folders, the CI will copy any scripts over to the other mission files. We only need to edit scripts in the main Altis mission folder. You can leave this in, it'll just get overwritten.